cirq-google/cirq_google/engine/engine_validator.py

```diff
@@ -83,30 +81,18 @@ def validate_program(
         repetitions:  Number of repetitions to run with each sweep.
         serializer:  Serializer to use to serialize the circuits and sweeps.
         max_size:  proto size limit to check against.
-
     Raises:
         RuntimeError: if compiled proto is above the maximum size.
     """
```

Comment:
It looks like validation was removed entirely. I think we still want to serialize and validate the resulting size.

Reply:
Added back!